contracts/base/Settings.sol

```diff
@@ -359,6 +361,25 @@ contract Settings is SettingsInterface, TInitializable, Pausable, BaseUpgradeabl
         );
     }
 
+    /**
+        @notice It sets the cToken address for a specific asset address.
+        @param assetAddress asset address to configure.
+        @param cTokenAddress the new cToken address to configure.
+     */
+    function _setCTokenAddress(address assetAddress, address cTokenAddress) internal {
+        if (assetAddress == ETH_ADDRESS) {
+            cTokenAddress.requireEqualTo(0x4Ddc2D193948926D02f9B1fE9e1daa0718270ED5, "CETH_ADDRESS_NOT_MATCH");
```

Comment:
Yea not really sure how we want to go about updating the ctoken addresses since I mean they shouldn't really ever need to change unless something catastrophic happens to Compound and they deploy new token addresses..